### PR TITLE
docs(pie-checkbox-group): DSW-2225 migrate checkbox code page

### PR DESCRIPTION
**Pull request by kevinrodrigues**
## Describe your changes (can list changeset entries if preferable)

- [Added] - Code pages to pie-doc site

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview

### Reviewer 1 - @xander-marjoram 
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview

### Reviewer 2
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview

### CODE REVIEW

File: apps/pie-docs/src/components/checkbox-group/overview/code.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+For Native JS Applications:
+```js
+// import as module into a js file e.g. main.js
+import '@justeattakeaway/pie-webc/components/checkbox-group.js'
+```
+
+```html
+<!-- pass js file into <script> tag -->
+<pie-checkbox-group>
```
Comment (kevinrodrigues) [inline apps/pie-docs/src/components/checkbox-group/overview/code.md:90]: One of the ACs in the ticket asks for form related examples. I think the below is sufficient enough for users but if we need more I can add further examples.

**Comment by leksaBoiko** [comment]
Just a suggestion, It might be helpful to provide a description on how to use a checkbox group with a group label? 
(similar to a text-iput page https://pr1672-docs.pie.design/components/text-input/code/#labelling)

```
 <PieCheckboxGroup>
      <PieFormLabel slot="label">Choose the way we can contact you:</PieFormLabel>
      <PieCheckbox
          checked={contactByEmail}
          onChange={handleContactByEmail}>
          Contact By Email
      </PieCheckbox>
      <PieCheckbox
          checked={contactByPhone}
          onChange={handleContactByPhone}>
          Contact By Phone
      </PieCheckbox>
  </PieCheckboxGroup>
```

**Review by ashleynolan — DISMISSED** [review]

File: apps/pie-docs/src/components/checkbox-group/overview/code.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  <pie-checkbox name="my-checkbox-three">Checkbox Label 3</pie-checkbox>
+</pie-checkbox-group>
+<script type="module" src="/main.js"></script>
+```
+
+For Vue Applications:
+
+```js
+// import as module into a js file that will be loaded on the page where the component is used.
+import '@justeattakeaway/pie-webc/components/checkbox-group.js';
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/checkbox-group/overview/code.md:102]: I think adding the checkbox import here too would be good

File: apps/pie-docs/src/components/checkbox-group/overview/events.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "headings": [
+    "Event",
+    "Type",
+    "Description"
+  ],
+  "rows": [
+    [
+      "`pie-checkbox-group-disabled`",
+      "`CustomEvent`",
+      "Triggered after the disabled state of the checkbox group changes."
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/checkbox-group/overview/events.json:11]: Can this event be heard outside of the checkbox group? I think the current implementation fires the event directly at/inside the children, which are listening for it.

Comment (kevinrodrigues) [reply apps/pie-docs/src/components/checkbox-group/overview/events.json]: As discussed have removed now.

File: apps/pie-docs/src/components/checkbox-group/overview/code.md (hunk 2 above)
Comment (kevinrodrigues) [reply apps/pie-docs/src/components/checkbox-group/overview/code.md:96]: Fixed.

**Review by xander-marjoram — APPROVED** [review]

**Review by JoshuaNg2332 — APPROVED** [review]

**Pull request merged**